RealFlight/Released_Models/QuadPlanes/Tailsitters/Addiction_3D/README.md

```diff
@@ -0,0 +1,35 @@
+# Addiction-X QuadPlane
+
+Aircraft Model:
+3D Model By: Erik Peterson - AKA: Keylitho - 
+
+
+![JPG](https://github.com/ArduPilot/SITL_Models/raw/master/RealFlight/Released_Models/QuadPlanes/Tailsitters/Addiction_3D/Addiction.jpg)
+
+Model modifications by : Andrew Tridgell
+
+
+* Wing Span....... 23 (in)
```

Comment:
the RealFlight physics for this model has half-span of 585mm which equates to a full span of 46"
But the aileron length is 640mm ???  probably should be 580mm